src/meta/src/hummock/vacuum.rs

```diff
@@ -287,7 +287,7 @@ where
     let workers = vec![
         cluster_manager.list_active_streaming_compute_nodes().await,
         cluster_manager
-            .list_worker_node(WorkerType::Compactor, Some(Running))
+            .list_worker_node(WorkerType::Compactor, Some(vec![Running, Cordoned]))
```

Comment:
Cordon should not be on the same level as running; it should be a flag.

Reply:
Resolved. Using `is_schedulable` flag for now. May switch to a negated `is_unschedulable`. [See discussion above. ](https://github.com/risingwavelabs/risingwave/pull/10143#discussion_r1227613598)